src/status_im2/contexts/syncing/events.cljs

```diff
@@ -72,14 +72,31 @@
                          :event    :syncing/input-connection-string-for-bootstrapping}))))]
     (native-module/prepare-dir-and-update-config "" default-node-config-string callback)))
 
+;; TODO :: Password verification is currently based on length but we have to do it realtime
+;; (rf/defn verify-user-password
+;;   {:events [:verify-profile-password]}
+;;   (println "In verify")
+;;   (fn [{:keys [address key-uid callback masked-password]}]
+;;     (let [hashed-password
+;;           (-> masked-password
+;;               security/safe-unmask-data
+;;               ethereum/sha3)]
+;;       (native-module/verify
+;;        address
+;;        hashed-password
+;;        (fn [result]
+;;          (let [{:keys [error]} (types/json->clj result)]
+;;            (log/info "[verify-profile-password] verify-password" result error)
+;;            (safe-blank? error)))))))
+
 (rf/defn preparations-for-connection-string
   {:events [:syncing/get-connection-string-for-bootstrapping-another-device]}
   [{:keys [db]} entered-password set-code]
   (let [valid-password? (>= (count entered-password) constants/min-password-length)
         show-sheet      (fn [connection-string]
                           (set-code connection-string)
                           (rf/dispatch [:syncing/update-role constants/local-pairing-role-sender])
-                          (rf/dispatch [:bottom-sheet/hide]))]
+                          (rf/dispatch [:hide-bottom-sheet]))]
```

Comment:
@smohamedjavid this is the right call for bottom sheet methods? 🤔

Reply:
`:bottom-sheet/hide` is an old bottom sheet code & event - this needs to be cleaned up with this implementation of authentication.

`:hide-bottom-sheet` - new bottom sheet event and it's correct.